lib/PhasarLLVM/DataFlow/IfdsIde/Problems/IFDSTaintAnalysis.cpp

```diff
@@ -404,7 +408,42 @@ auto IFDSTaintAnalysis::getSummaryFlowFunction([[maybe_unused]] n_t CallSite,
       Kill.insert(SRet);
     }
   }
+  if (Gen.empty() && Leak.empty() && Kill.empty()) {
+    if (Llvmfdff.contains(DestFun)) {
+      return lambdaFlow([this, CallSite,
+                         DestFun](d_t Source) -> container_type {
+        std::set<d_t> Facts;
+        // replace with for loop
+        const auto *CS = llvm::cast<llvm::CallBase>(CallSite);
+        int Index = 0;
+        for (const auto *Iter = CS->data_operands_begin();
+             Iter != CS->data_operands_end(); ++Iter) {
+          // if (const auto *Arg = llvm::dyn_cast<llvm::Argument>(Source)) {
+          if (Source == llvm::dyn_cast<llvm::Value>(Iter)) {
+            auto VecFacts = Llvmfdff.getFacts(DestFun, DestFun->getArg(Index));
```

Comment:
Can we somehow compute this (or at least parts of this) outside of the lambda and capture its result, suc that it does not have to be re-computed every time?